### ISSUE TITLE
Spotify Extension

### ISSUE THREAD

**Issue by Scratchfangs** [TurboWarp/extensions#274]
Hello there! I believe it would be a good idea to make a Spotify extension like the one from ScratchX that plays about 30 second previews of songs and can detect beats.

https://ericrosenbaum.github.io/spotify-extension/

**Comment by TheShovel** [comment]
As far as I know, it does work, you have if you run it unsandboxed.

**Comment by Scratchfangs** [comment]
> As far as I know, it does work, if you run it unsandboxed.

I've tried running in Turbowarp, at the time it's not working. It did work a couple of months ago, however.

**Comment by True-Fantom** [comment]
I'll probably refine it a bit and make an add request (In the next 3 days)

**Comment by True-Fantom** [comment]
I am having some problems with the code.  It looks like this won't work properly.

**Comment by Scratchfangs** [comment]
Yep, that is exactly what I noticed, I don't know what seems to be the problem though...

**Comment by BlueDome77** [comment]
There’s a Spotify extension working here: https://scratch.machinelearningforkids.co.uk/

**Comment by BlueDome77** [comment]
https://github.com/TurboWarp/extensions/pull/1143
